elixir/scripts/test_emails.exs

```diff
@@ -0,0 +1,344 @@
+# Email Testing Script for IEx
+#
```

Comment:
I've kept this helper script in a separate commit so it's easy to drop it if you think it's not worth keeping (also; wasn't sure where to put it!).

I personally found value in being able to quickly send all the emails and inspect them, but I might just not know how to do it the "proper" way in Elixir :)